appcues/src/main/java/com/appcues/data/AppcuesRepository.kt

```diff
@@ -148,13 +150,21 @@ internal class AppcuesRepository(
 
             qualifyResult.doIfSuccess { response ->
                 val priority: ExperiencePriority = if (response.qualificationReason == "screen_view") LOW else NORMAL
-                experiences += response.experiences.map { experienceMapper.map(it, priority, response.experiments, activity.requestId) }
+                experiences += response.experiences.filterIsInstance<ExperienceResponse>().map {
```

Comment:
`filterIsInstance<ExperienceResponse>` here is where we are just throwing out the experiences that failed to deserialize at this point, during mapping to domain objects.